### ISSUE TITLE
Warnings and errors when building docs

### ISSUE THREAD

**Issue by dneuman64** [Comcast/traffic_control#1385]
There are several errors and warning that are printed when docs are built.  These should be fixed.  

Go to the traffic_control/docs dir. 
Type `make html`
The following errors and warnings are shown: 

```
~/git/traffic_control/docs/source/admin/traffic_router.rst:167: ERROR: Unexpected indentation.
~/git/traffic_control/docs/source/admin/traffic_router.rst:251: ERROR: Unexpected indentation.
~/git/traffic_control/docs/source/admin/traffic_router.rst:282: ERROR: Unexpected indentation.
~/git/traffic_control/docs/source/admin/traffic_router.rst:323: WARNING: Title underline too short.

GeoLimit Failure Redirect feature
======
~/git/traffic_control/docs/source/admin/traffic_router.rst:323: WARNING: Title underline too short.

GeoLimit Failure Redirect feature
======
~/git/traffic_control/docs/source/admin/traffic_router.rst:337: WARNING: Title underline too short.

Configuration
--------
~/git/traffic_control/docs/source/admin/traffic_router.rst:337: WARNING: Title underline too short.

Configuration
--------
~/git/traffic_control/docs/source/admin/traffic_router.rst:342: ERROR: Unexpected indentation.
~/git/traffic_control/docs/source/admin/traffic_router.rst:343: WARNING: Block quote ends without a blank line; unexpected unindent.
~/git/traffic_control/docs/source/admin/traffic_stats.rst:222: WARNING: Bullet list ends without a blank line; unexpected unindent.
~/git/traffic_control/docs/source/admin/traffic_stats.rst:229: WARNING: Definition list ends without a blank line; unexpected unindent.
~/git/traffic_control/docs/source/development/traffic_ops_api/v12/user.rst:413: WARNING: Block quote ends without a blank line; unexpected unindent.
~/git/traffic_control/docs/source/faq/general.rst:32: WARNING: Definition list ends without a blank line; unexpected unindent.
~/git/traffic_control/docs/source/faq/general.rst:18: WARNING: Duplicate explicit target name: "here".
```

**Comment by sradvan** [comment]
@dneuman64 it should build without errors now

**Comment by dneuman64** [comment]
sweet, thanks @sradvan!  I will give it a look when I get a chance

**Comment by dneuman64** [comment]
oh wait, @knutsel already merged.  this is closed by #1393
